src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+
+export default function Dropdown() {
+	const [drop, setdrop] = useState(true);
+	function dropMenu () {
+		setdrop(!drop);
+	}
+	let dropClass = drop ? "dropdown-content-off" : "dropdown-content";
```

Comment:
This should be `const` you don't mutate this variable afterwards in this function.